skimage/filters/tests/test_fft_based.py

```diff
@@ -0,0 +1,58 @@
+import pytest
+import os
+import numpy as np
+from .. import butterworth
+from skimage.data import astronaut, coins
```

Comment:
```suggestion
from skimage.data import astronaut, coins
from .. import butterworth
```

Reply:
Actually, 

```suggestion
from skimage.data import astronaut, coins
from skimage.filters import butterworth
```

We use absolute imports in our tests.